### PR TITLE
Upcoming VEX post

### PR DESCRIPTION
**Pull request by hayleycd**
Forum for discussion: #1080 

Author: @lumjjb

### CODE REVIEW

File: gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: "Automating and Scaling Vex Generation"
+date: 2023-03-03T16:46:51Z
+draft: true
+author: Brandon Lum
```
Comment (lumjjb) [inline gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md:5]: are we able to add "Oliver Chang" and "Meder Kydyraliev" as well?

**Review by lumjjb — APPROVED** [review]

File: gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md (hunk 1 above)
Comment (hayleycd) [reply gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md:5]: We certainly can!

Comment (hayleycd) [inline gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md:5]:
```suggestion
author: Brandon Lum, Oliver Chang, and Meder Kydyraliev
```

File: gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+title: "Automating and Scaling Vex Generation"
+date: 2023-03-03T16:46:51Z
+draft: true
+author: Brandon Lum, Oliver Chang, and Meder Kydyraliev
```
Comment (hayleycd) [inline gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md:5]: Please confirm that looks ok to you @lumjjb

File: gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+---
+title: "Automating and Scaling Vex Generation"
+date: 2023-03-03T16:46:51Z
```
Comment (hayleycd) [inline gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md:3]:
```suggestion
date: 2023-03-06T09:00:00Z
```

File: gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
+---
+title: "Automating and Scaling Vex Generation"
+date: 2023-03-06T09:00:00Z
+draft: true
```
Comment (hayleycd) [inline gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md:4]: I picked a time, but we can adjust it if necessary. 
"draft" must also be changed to "false"

Comment (hayleycd) [inline gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md:4]:
```suggestion
draft: false
```

File: gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
+---
+title: "Automating and Scaling Vex Generation"
+date: 2023-03-06T09:00:00Z
+draft: false
```
Comment (hayleycd) [inline gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md:4]: I picked a time, but we can adjust it if necessary. 
I am setting "draft" to "false". 

When this is merged, this should be considered live.

**Comment by hayleycd** [comment]
@andrewpollock @oliverchang @lumjjb 

Please include this in the upcoming frontend release. 

This is set as no longer a draft. 

We might want to check this on stage first if we are concerned about the images.

**Review by andrewpollock — APPROVED** [review]

**Pull request merged**